.github/workflows/Crane.yaml

```diff
@@ -53,10 +53,16 @@ jobs:
             ~/.gradle/caches/build-cache-*
           key: gradle-${{ hashFiles('checksum.txt') }}
 
+    - name: Inject Maps API Key
+      run: |
+        # Injecting the key directly into the manifest as secrets-gradle-plugin
+        # isn't picking up the key when creating a local.properties file here
+        sed -i -e "s,\${googleMapsKey},$GOOGLE_MAPS_KEY,g" ${{ env.SAMPLE_PATH }}/app/src/main/AndroidManifest.xml
+      env:
+        GOOGLE_MAPS_KEY: ${{ secrets.GOOGLE_MAPS_KEY }}
```

Comment:
This is great - Would it also fix #746 ? CityMapViewTests

Reply:
This uses the same API key so if the issue is due to key restrictions this won't fix it. I can take a look at #746  though to see what's going on